src/test/java/uk/gov/ons/census/action/schedule/CaseClassifierTest.java

```diff
@@ -44,4 +47,38 @@ public void testEnqueueCasesForActionRule() {
     verify(jdbcTemplate)
         .update(eq(expectedSql.toString()), any(UUID.class), eq(actionRule.getId()));
   }
+
+  @Test
+  public void testEnqueueCeEstabCasesForActionRule() {
+    // Given
+    JdbcTemplate jdbcTemplate = mock(JdbcTemplate.class);
+
+    CaseClassifier underTest = new CaseClassifier(jdbcTemplate);
+    Map<String, List<String>> classifiers = new HashMap<>();
+    classifiers.put("treatment_code", List.of("abc", "xyz"));
+    ActionPlan actionPlan = new ActionPlan();
+    actionPlan.setId(UUID.randomUUID());
+    ActionRule actionRule = new ActionRule();
+    actionRule.setId(UUID.randomUUID());
+    actionRule.setActionPlan(actionPlan);
+    actionRule.setClassifiers(classifiers);
+    actionRule.setActionType(ActionType.CE_IC03);
+
+    // When
+    underTest.enqueueCasesForActionRule(actionRule);
+
+    // Then
+    StringBuilder expectedSql = new StringBuilder();
+    expectedSql.append("INSERT INTO actionv2.case_to_process (batch_id, batch_quantity, ");
+    expectedSql.append(
+        "action_rule_id, caze_case_ref, ce_expected_capacity) SELECT ?, SUM(ce_expected_capacity) OVER(),"
+            + " ?, case_ref, ce_expected_capacity ");
```

Comment:
you could tidy up these append statements so each one isn't longer than 100 chars... make it a bit neater and readable